src/Database/DBA.php

```diff
@@ -1459,7 +1459,7 @@ public static function buildParameter(array $params = [])
 
 		$limit_string = '';
 		if (isset($params['limit']) && is_int($params['limit'])) {
-			$limit_string = " LIMIT " . $params['limit'];
+			$limit_string = " LIMIT " . intval($params['limit']);
```

Comment:
The variable type is already ensured in the conditional, no need to typecast.

Reply:
I know. This is like wearing braces when you already wears a belt. But it looks more symmetrical to the lines below where we are doing the same.